### DISCUSSION TITLE
Matplotlib and VTK support for VCL and/or FMX

### DISCUSSION THREAD

**Discussion by dewloosh** [Embarcadero/DelphiVCL4Python#34] [Ideas]
Hi there!

I just want to start this discussion about VCL/FMX and Matplotlib/VTK. In my opinion, until there is no viable backend for these, VCL/FMX cannot be a real competitor for Qt.

**Comment by Priyatham10** [comment]
Hi,

Yeah, that's absolutely true, and we have that in our plans. However, we're currently occupied by the process of adding the API documentation for both `delphivcl` and `delphifmx` packages. Any ideas and research on implementing the Matplotib blackened until then are highly appreciated.

**Comment by lmbelo** [comment]
Agree. We must put some work on it after docs gets done. Any help would be appreciated.

**Reply by xichaoqiang** [reply]
How is the current development progress going?

**Reply by lmbelo** [reply]
We are running other priorities by now. But this is something we are looking after.

**Comment by Bollebie** [comment]
is there any progress on showing matplotlib plots in the Gui window and live update the plot
